Libplanet/Store/IStore.cs

```diff
@@ -53,6 +53,26 @@ public interface IStore
 
         bool DeleteIndex(string @namespace, HashDigest<SHA256> hash);
 
+        /// <summary>
+        /// Forks block indexes from
```

Comment:
Does the term “block indexes” mean a sequence (chain) of block hashes and its index numbers?  Or does it include state references?  If it's latter the plural term seems natural for me as well, but if it's former I believe the singular term (i.e., “block index”) would listen more natural IMHO (as like the way other related methods are named, e.g., `AppendIndex`, `DeleteIndex`, `IterateIndex`).

Reply:
It's former. but I've confused about that because….

- `AppendIndex()` and `DeleteIndex()` take only one argument. 
- Other iteration method use plural form. (e.g. `IterateBlockHashes()`, `IterateStateReferences()`)

so, How about keep `ForkIndexes()` in this PR and rename `IterateIndex()` to `IterateIndexes()` in other PR?

Reply:
IMO in the case of this a singular noun “index” is more natural than a plural noun “indexes”/“indices,” in the same manner we say “a set of blocks” instead of “sets of blocks” to mean `ISet[Block]` in verbal language.

Reply:
I thought about this terminology once more, and it turns out the plural term has its own advantage, and I agree your suggestion to rename these method names to use the plural term “indexes” instead of the singular “index.”  Some reasons:

- Although relational databases tend to say “an index” for values in a column, we also say “indices” in books.
- We already name a block's height by `Block.Index`, so it's natural to say a collection of `Block.Index`es as the plural “indexes.”